HW_4/task_3.py

```diff
@@ -0,0 +1,28 @@
+# Задать натуральное число k.
+# Сформируйте многочлен (полином) степени k со случайными коэффициентами из промежутка от 0 до 100, включительно.
+# Многочлен вывести в консоль и записать в файл.
+
+
+import random
+
+k = int(input('Введите число k = '))
+coef = [random.randint(-100, 100) for i in range(0, k + 1)]
```

Comment:
Вообще, нужно было брать коэффициенты из диапазона от 0 до 100. Хотя в вашем случае я нисколько не против.
```suggestion
coef = [random.randint(-100, 100) for i in range(k + 1)]
```